monitorlog.md

```diff
@@ -1,66 +1,99 @@
-* [User Data Access Logging](#user-data-access-logging)
+<br>[Logging with LogDNA](#log-files-with-logdna)
+<br>[About Management API monitoring tools](#about-management-api-monitoring-tools)
+<br>[Using the New Relic synthetic monitor](#new-relic-synthetic-monitor)
+<br>[About user data access logging](#about-user-data-access-logging)
 
-## Log files with LogDNA  
-[HRI](glossary.md#hri) uses the [IBM Cloud LogDNA](https://cloud.ibm.com/docs/Log-Analysis-with-LogDNA) service for application logging purposes. There are two sources of logs within the HRI: the Management API (IBM Functions) and the Document Store [(Elastic Search)](glossary.md#elasticsearch). Both send logs to the LogDNA instance designated for "Platform Logs" in your particular IBM Cloud Region for your [IBM Cloud Functions](glossary.md#ibm-cloud-functions) account. 
-   
-You click on the "View LogDNA" button for the LogDNA instance associated with your Resource Group located in the region where your IBM Cloud Functions instance is located, e.g. `Dallas(US-South)`.
+## Logging with LogDNA  
+For application logging purposes, Health Record Ingestion uses the IBM&reg; Cloud LogDNA service. In Health Record Ingestion, there are two sources of logs: 
 
-### Using LogDNA to View HRI Logs
-Once you have opened the correct instance of LogDNA, you will likely want to use the filtering feature to be able to identify the logs of interest to you. First you may want to filter by Source. There are 3 filter drop-down menus you can choose at the top of your screen. Choose the "All Sources" menu and select the "functions" checkbox to filter only on IBM Functions log messages as seen in this Screen Cap: 
+- The [Management API ](glossary.md#management-api) (IBM Functions) 
+- The Document Store [(Elastic Search)](glossary.md#elasticsearch)
+
+Both send logs to the LogDNA instance designated for Platform Logs in your particular IBM Cloud Region for your [IBM Cloud Functions](glossary.md#ibm-cloud-functions) account. For more information about LogDNA, see [IBM Cloud LogDNA](https://cloud.ibm.com/docs/Log-Analysis-with-LogDNA)
+
+**To use LogDNA:**
+
+1) Open the correct instance of LogDNA. Use the LogDNA instance associated with your Resource Group. Use the group that's located in the region where your IBM Cloud Functions instance is located, for example, `Dallas(US-South)`.
+2) Click **View LogDNA**. 
+
+
+### Using LogDNA to view Health Record Ingestion logs
+After you open the correct instance of LogDNA, you'll typically use filtering to find logs of interest. 
+
+**To filter logs:**
+ - Try filtering by source. Select from the menus in the menu bar (Figure 1). 
+
+Here are examples:
+- To filter for IBM Functions log messages, go to the **All Sources** menu and select **functions**.
+- To filter for Elastic Search logs, you would select `ibm-cloud-database-prod`. 
+- To filter to a specific instance, go to **All Apps** menu and use the Cloud Resource Name (CRN) identifier. 
```

Comment:
Not sure this really works. Make include the second list as ways to filter for specific logs.